src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -309,6 +309,7 @@ public SendReceive(ITestOutputHelper output)
         [OuterLoop] // TODO: Issue #11345
         [Theory]
         [MemberData(nameof(Loopbacks))]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/23364", TargetFrameworkMonikers.UapAot)]
```

Comment:
nit:  we usually just use numbers instead of strings if the issue is in the same repo.